parsel/selector.py

```diff
@@ -339,8 +339,8 @@ def remove_namespaces(self):
             for an in el.attrib.keys():
                 if an.startswith('{'):
                     el.attrib[an.split('}', 1)[1]] = el.attrib.pop(an)
-            # remove namespace declarations
-            etree.cleanup_namespaces(self.root)
+        # remove namespace declarations
+        etree.cleanup_namespaces(self.root)
```

Comment:
How does this affect XML documents embedded in other XML documents? (e.g. SVG within HTML)

If it works nonetheless, maybe we should extend the test suite to cover that case.

Reply:
Added a test for a SVG file embedded in a XML one (based on https://developer.mozilla.org/es/docs/Web/SVG/Tutorial/SVG_In_HTML_Introduction#Source), according to the namespaces specified in https://www.w3.org/TR/html5/infrastructure.html#namespaces. The same test works with the current code as well, no changes there.